docs/how-to-work-on-practice-projects.md

```diff
@@ -1,18 +1,74 @@
 # How to Work on Practice Projects
 
+Our practice projects use a step-based approach to teach concepts to campers. A project will consist of multiple files, which we refer to as "steps". These files are named by the challenge ID, to avoid issues with the translation flow. Unfortunately, this makes it difficult to find the file associated with "step 3", for example.
```

Comment:
```suggestion
Our practice projects use a step-based approach to teach concepts to campers. A project will consist of multiple files, which we refer to as **"steps"**. These files are named by the challenge ID, to avoid issues with the translation flow. Unfortunately, this makes it difficult to find the file associated with a specific step.
```